operator/README.md

```diff
@@ -44,7 +44,7 @@ Make sure KUBECONFIG is set:
 ```sh
 make admin.kubeconfig
 export KUBECONFIG=$(pwd)/admin.kubeconfig
-oc delete -n openshift-azure-operator deployment/aro-operator
+oc scale -n openshift-azure-operator deloyment/aro-operator-master --replicas=0
```

Comment:
this will make it easier to resume the in-cluster pod. 
we don't need to stop both master/worker, since locally we always run master version.